grails-app/assets/stylesheets/elurikkus-common.css

```diff
@@ -280,13 +280,13 @@ input:active, textarea:active, select:active {
     color: inherit;
 }
 
-.erk-button:hover, .erk-button: active, .erk-button:active {
+.erk-button:hover, .erk-button:active, .erk-button:active {
```

Comment:
Aren't those ``erk-button:active`` duplicates?

Reply:
Looks like it. This whole thing wasn't working anyway because of the extra space